app/assets/stylesheets/components/_contact-history.scss

```diff
@@ -1,5 +1,8 @@
-.contact-history {
+#contact-history {
```

Comment:
We made this an ID to make it easier to select the element and replace it with data from the server.